test/e2e/e2e_suite_test.go

```diff
@@ -320,6 +352,9 @@ func newCPIInstallValues() map[string]interface{} {
 			"image": image,
 			"tag":   version,
 		},
+		"securityContext": map[string]interface{}{
```

Comment:
We don't put it inside our helm yaml?

Reply:
They why do we need it here?

Reply:
We need it because `securityContext` is not required for e2e, so having this can override the value defined here https://github.com/kubernetes/cloud-provider-vsphere/blob/b3ee1f01a52092a99bcba00393c8174fd1264e22/charts/vsphere-cpi/values.yaml#L42